stingray/gti.py

```diff
@@ -97,6 +154,28 @@ def get_gti_extensions_from_pattern(lchdulist, name_pattern="GTI"):
     return gtiextn
 
 
+def hdu_contains_gti(hdu):
+    """Test if a given hdu contains a list of GTIs
+
+    Examples
+    --------
+    >>> from astropy.io import fits
+    >>> start = np.arange(0, 300, 100)
+    >>> stop = start + 50.
+    >>> s1 = fits.Column(name='START', array=start, format='D')
+    >>> s2 = fits.Column(name='STOP', array=stop, format='D')
+    >>> hdu1 = fits.TableHDU.from_columns([s1, s2], name='BLABLA')
+    >>> hdu_contains_gti(hdu1)
+    True
+    >>> s2 = fits.Column(name='blabla', array=stop, format='D')
+    >>> hdu1 = fits.TableHDU.from_columns([s1, s2], name='BLABLA')
+    >>> hdu_contains_gti(hdu1)
+    False
+    """
+    colnames = [c.lower() for c in hdu.data.columns.names]
+    return "start" in colnames and "stop" in colnames
```

Comment:
Are we sure that there are no other reasons why the words "start" and "stop" could occur in FITS columns?

Reply:
I'm not 100% sure, but I have never seen columns START and STOP in a FITS file outside the GTI extension